### ISSUE TITLE
Unexpected drag-and-drop behavior of .zarr into napari

### ISSUE THREAD

**Issue by talonchandler** [mehta-lab/recOrder#82]
**Summary**
I'm observing unexpected drag-and-drop behavior of .zarr folders into napari. I'm looking for context: is this expected behavior? a bug in the recOrder CLI converter? or my misuse of .zarr+napari? Thanks for your help!

**Details**
Hi @smguo @mattersoflight @JohannaRahm @jrbyrum13. One of the action items from our meeting this afternoon was to check that multiprocessing doesn't mangle the zarr metadata. 

To check this I started by trying to open the "raw" zarr data (in quotes since I think this has been converted from the truly raw tifs to zarr using the recOrder CLI), and I ran into some issues.
When I started a napari interactive session on cpu-b-001 I dragged and dropped the top-level zarr (`/hpc/projects/compmicro/rawdata/hummingbird/Janie/2022_03_10_orgs_nuc_mem_63x_04NA/1_12_pos_1.zarr`) into napari I received an error: `'list' object has no attribute 'shape'`:
![Screen Shot 2022-03-24 at 5 37 12 PM](https://user-images.githubusercontent.com/9554101/160031973-bd58d974-c896-4e60-8d57-adbb3cad441d.png)

I could only view the data one position at a time when I dragged in the lowest level arr_0 folder. 

Surprisingly, if I started napari with the shared recOrder conda environment then dragged the same top-level folder in I received different behavior: a very wide image like this
![Screen Shot 2022-03-24 at 5 37 54 PM](https://user-images.githubusercontent.com/9554101/160032748-34eab08b-fa6a-4519-b8c3-d51ab68f76d2.png)

**Expected behavior**
I'm guessing that neither of these screenshots are expected behaviors. I'm expecting to be able to drag in a zarr folder and have z slices on one slider, channels on a second slider, and positions on a third slider. 

**Environment:**
Napari interactive session via OnDemand without recOrder environment (first screenshot) and with recOrder environment (second screenshot)

Operating system: XFCE
Python version: 3.7.11
Python environment (command line, IDE, Jupyter notebook, etc): command line, using shared recOrder environment on HPC
MicroManager/pycromanager version: not sure...might be able to find in metadeta of Janie's 2022_03_10 data.

**Comment by talonchandler** [comment]
@mattersoflight and I discussed in person. Dragging and dropping the individual positions worked as expected, but dragging the whole zarr was still a problem. We think we traced the issue to a problem with the tiff->zarr conversion. I will follow up.

**Comment by camFoltz** [comment]
@talonchandler I did try to implement a custom napari reader that allows the user to drag and drop the root zarr store, however, this was a later feature that didn't get thoroughly tested.

I would however be quite surprised if it were related to the tiff --> zarr conversion.  If we assume the reader that I implemented is broken, then I am not aware of another reader that is able to produce the behavior your expecting.

I would suggest updating the reader to work more cleanly (I think the biggest issue is napari being unable to recognize it as a reader).

Here is the relevant code: [recOrder napari Reader](https://github.com/mehta-lab/recOrder/blob/main/recOrder/io/_reader.py#L12)

Happy to discuss further if needed.

**Comment by jrbyrum13** [comment]
Hi Talon and Cam, just noting that I run into this issue, too. I can only
drop position-level zarr images.

**Comment by talonchandler** [comment]
Thanks for the pointers @camFoltz, and thanks for confirming @jrbyrum13. 

I'll dig into the reader implementation and see if I can fix this.

**Comment by talonchandler** [comment]
I just tested this, and it is now fixed. I strongly suspect that this was a packaging error that led `napari` to use to incorrect readers, as Cameron suggested.  

I just tested a drag and drop of this file:
`/hpc/projects/compmicro/rawdata/hummingbird/Janie/2022_03_10_orgs_nuc_mem_63x_04NA/1_12_pos_1.zarr`,

and I confirmed that I can open multi-position zarrs one using `recOrder`'s main branch and the hpc's shared `recOrder` environement.

Closing.
